WcaOnRails/app/models/competition.rb

```diff
@@ -1681,6 +1682,22 @@ def set_wcif!(wcif, current_user)
     end
   end
 
+  def set_wcif_competitor_limit!(wcif_competitor_limit, current_user)
+    unless current_user.can_admin_competitions? || !confirmed?
```

Comment:
I think you should separate these two checks to provide clear error messages.

if not admin: "Cannot edit the competition."
if confirmed: "Cannot edit the competitor limit because the competition has been confirmed by WCAT."

Reply:
Thanks! I updated the error message to `"Cannot edit the competitor limit because the competition has been confirmed by WCAT."`. However, I originally intended this as a single check: if it's confirmed, and the person isn't an admin, raise the error. This would let the WCAT change the competitor limit after a comp is confirmed. Does this seem reasonable?

As a follow-up, I got a big confused about user permissions: do I need to perform a separate check here? If so, is `admin` the correct check, or should I be doing `can_manage_competitions` or something else?